examples/blueprint-construct/index.ts

```diff
@@ -74,6 +75,19 @@ export default class BlueprintConstruct {
             new blueprints.addons.CoreDnsAddOn(),
             new blueprints.addons.KubeProxyAddOn(),
             new blueprints.addons.OpaGatekeeperAddOn(),
+            new blueprints.addons.AckAddOn({
+                skipVersionValidation: true
+            }),
+            new blueprints.addons.AckAddOn({
+                skipVersionValidation: true,
+                name: "rds-chart",
+                chart: "rds-chart",
+                version: "v0.1.1",
```

Comment:
is this the latest version?

Reply:
Yes. You can check the ECR link https://gallery.ecr.aws/aws-controllers-k8s/ for all 47 charts.